sigs.yaml

```diff
@@ -174,6 +201,63 @@ sigs:
         teams:
         - name: kubevirt-community
           description: General Discussion
+      subprojects:
+      - name: ci-s390x
```

Comment:
@dhiller 
A Question fust for my understanding-
Currently it seems these subproject names aren't used in any OWNERS files. So do we need to update/create OWNERS file along side the code files/dirs while we commit the code and hence we don't refer sub-project names anywhere?

Reply:
yes, exactly - my intent is to add the subprojects first, and when you add new OWNERS files where you take ownership of code, then the sigs.yaml will be updated with a reference (hopefully by some automation :smile: )

Reply:
when you say "the sigs.yaml will be updated with a reference ", do you mean updating the "owners:" section in the subprojects of sigs.yaml?

Reply:
yes